packages/web-embed-api/src/index.ts

```diff
@@ -36,7 +36,12 @@ export interface IEventMap {
   flowloaded?: () => void;
   flowclosed?: () => void;
   flowfinalized?: () => void;
-  redirect?: (p: string) => void;
+  redirect?: (
+    p: string
```

Comment:
While we're introducing non-backward compatible changes, let's make the signature `({ url: string })` instead of `(p: string)`? 

- Not sure why it was ever named `p` :)
- As an object parameter, we can add in a followup also sending `answers` (if embedded in a whitelisted domain) and other properties in here.

BTW I don't think we should actually _publish_ 2.0 until we wrap up a few more of these changes together

Reply:
Sounds good, will change to `({ url: string })`